crates/miden-lib/asm/account_components/rpo_falcon_512_procedure_acl.masm

```diff
@@ -62,10 +70,44 @@ export.auth__tx_rpo_falcon512_procedure_acl
         dup.1 neq.0
         # => [should_continue, was_any_called', i-1, pad(16)]
     end
-
     # => [was_any_called, i-1, pad(16)]
 
-    # If any trigger procedure was called, perform signature verification
+    # ------ Check if output notes were created ------
+
+    exec.tx::get_output_notes_commitment
+    # => [OUTPUT_NOTES_COMMITMENT, was_any_called, i-1, pad(16)]
```

Comment:
Nit: I don't think we need `i-1` anymore after the loop, right? So should we just drop it here to avoid having to pass it through in every stack state comment?